examples/hacker-news/client/containers/LoginInfo.js

```diff
@@ -1,14 +1,13 @@
 import React from 'react'
-import { connectReadModel } from 'resolve-redux'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
 import styled from 'styled-components'
-import { NavLink } from 'react-router-dom'
+import { Link as NormalLink } from 'react-router-dom'
 
 import Splitter from '../components/Splitter'
-import { connect } from 'react-redux'
 import * as userActions from '../actions/userActions'
-import { bindActionCreators } from 'redux'
 
-const Link = styled(NavLink)`
+const Link = styled(NormalLink)`
```

Comment:
I guess it'll be better to have NormalLink as Link, and Link as '%Smth%Link' (for example. CustomLink or something like this)